python/ClusterLauncher/PBSJob.py

```diff
@@ -61,17 +61,23 @@ def copyFiles(self, job_file):
         for file in os.listdir('../'):
             if os.path.isfile('../' + file) and file != job_file and \
                (not params.isValid('no_copy') or file not in params['no_copy']) and \
-               (not params.isValid('no_copy_pattern') or pattern.match(file) is None):
+               (not params.isValid('no_copy_pattern') or pattern.match(file) is None) and \
+               not os.path.exists(file) and \
+               os.path.splitext(file)[1] != '':
```

Comment:
do not copy anything without an extension by default. This can still be overridden with the copy_files routine below.